docs/source/protocols/433.92/switch/rev_v5.rst

```diff
@@ -0,0 +1,231 @@
+.. |yes| image:: ../../../images/yes.png
+.. |no| image:: ../../../images/no.png
+
+.. role:: underline
+   :class: underline
+
++------------------+-------------+
+| **Feature**      | **Support** |
++------------------+-------------+
+| Sending          | |yes|       |
++------------------+-------------+
+| Receiving        | |yes|       |
++------------------+-------------+
+| Config           | |yes|       |
++------------------+-------------+
+
+.. rubric:: Supported Devices
+
++----------------------+----------------------+
+| **Brand**            | **Protocol**         |
++----------------------+----------------------+
+| REV                  | Switch 8462          |
+|                      |                      |
+|                      | Remote 8471          |
++----------------------+----------------------+
+
+.. rubric:: Sender Arguments
+
+.. code-block:: console
+   :linenos:
+
+   -i --id=id             control a device with this id/code
+   -u --unit=unit         control a device with this unit/button
+   -a --all=1/2           control all devices with unitcode 0-3 (when -a 1) or unitcode 4-7 (when -a 2)
+   -t --on                send an on signal
+   -f --off               send an off signal
+
+   provide either --unit or --all
+
+.. Note:: usage of the all-parameter
+
+ - when using the --all parameter, the --unit parameter is ignored.
+ - the all-parameter needs a value, either 1 or 2
+ - when passing 1, all switches with unit 0 or 1 or 2 or 3 are switched to the provided state
+ - when passing 2, all switches with unit 4 or 5 or 6 or 7 are switched to the provided state
+
+
+.. rubric:: Config Sample 1
+
+.. code-block:: json
+   :linenos:
+
+   {
+     "devices": {
+       "radio": {
+         "protocol": [ "rev5_switch" ],
+         "id": [{
+           "id": "A",
+           "unit": 1
+         }],
+         "state": "off"
+       }
+     },
+     "gui": {
+       "radio": {
+         "name": "Kitchen radio",
+         "group": [ "Kitchen" ],
+         "media": [ "all" ]
+       }
+     }
+   }
+
+.. rubric:: Config Sample 2
+
+.. code-block:: json
+   :linenos:
+
+   {
+     "devices": {
+       "radio": {
+         "protocol": [ "rev5_switch" ],
+         "id": [{
+           "id": "A",
+           "unit": 1
+         }],
+         "all": 1 /*when all is specified, the unit value is ignored */
```

Comment:
JSON doesn't allow comments, so this most likely lead to frustration when copy pasting.